stdlib/public/runtime/Reflection.mm

```diff
@@ -313,71 +304,25 @@ void swift_MagicMirrorData_summary(const Metadata *T, String *result) {
   return swift_getObjCClassMetadata(isa);
 }
  
-static std::tuple<const _ReflectableWitnessTable *, const Metadata *,
-                  const OpaqueValue *>
+static std::tuple<const Metadata *, const OpaqueValue *>
 getReflectableConformance(const Metadata *T, const OpaqueValue *Value) {
```

Comment:
The name of this function no longer makes sense. How about just unwrapExistential()?